src/Squisher.js

```diff
@@ -93,13 +122,80 @@ class Squisher {
         }
         const entity = this.clickListeners[click.x * this.width + click.y];
         if (entity) {
-            entity.handleClick(player, translatedX, translatedY);
+            entity.handleClick && entity.handleClick(player, translatedX, translatedY);
         }
     }
+    
+    squish(entity) {
+        // Type (1) + Size (1) + color (4) + pos (4) + size (4) + text position (2) + text (32) + assets (37 * assetCount)
```

Comment:
This function is gross and I don't like it. I want to clean this up but I'm still making big changes to what's sent over the socket so I will wait until that's a little more clear before abstracting this logic